src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -90,6 +90,11 @@ export const terminalConfiguration: IConfigurationNode = {
 			type: 'boolean',
 			default: false
 		},
+		'terminal.integrated.altClickMovesCursor': {
+			description: localize('terminal.integrated.altClickMovesCursor', "If enabled, alt/option + click will move the prompt cursor to position underneath the mouse."),
```

Comment:
```suggestion
			description: localize('terminal.integrated.altClickMovesCursor', "If enabled, alt/option + click will move the prompt cursor to position underneath the mouse. This may not work reliably depending on your shell."),
```